Formula/luajit.rb

```diff
@@ -199,8 +203,8 @@ def install
  }
 @@ -143,7 +185,7 @@
  #endif
- 
- /* This function supports releasing coalesced segments */
+
+ /* This function supports releasing coalesed segments */
```

Comment:
Won't this change cause the patch to fail to apply?

Reply:
The patch is currently failing with the master version due to this PR: #66898

Reply:
Ugh, I see. Ok. I'm going to have to check that that didn't break any other patches.

Reply:
Apologies for that. I've reverted in https://github.com/Homebrew/homebrew-core/commit/b822aca874f3a39203fc7ec3c412399882feded9